example/package.json

```
@@ -21,13 +21,13 @@
     "react": "~16.9.0",
     "react-dom": "~16.9.0",
     "react-native": "~0.61.5",
-    "react-native-gesture-handler": "~1.5.3",
+    "react-native-gesture-handler": "~1.5.5",
```

can you please cosistenly use "^"? There's no reason for ~
```suggestion
    "react-native-gesture-handler": "^1.5.5",
```